presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -1113,6 +1113,64 @@ public void testRenameSchema()
     }
 
     @Test
+    public void testUnicodeString()
+            throws Exception
+    {
+        assertStatement("CREATE TABLE foo (a VARCHAR COMMENT U&'hello\\6d4B\\8Bd5\\+10FFFFworld\\7F16\\7801')",
```

Comment:
Use `assertExpression` since we're testing an expression. Doing a full, complex statement makes the test too broad.